docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -171,6 +171,20 @@ Search parameters are handled like a single configuration properly; providing a
 
 More information about multi-tenant support can be found in [Section 4.9 Multi-tenancy](#49-multi-tenancy).
 
+## 3.3.1 Compartment Search Performance
+
+The IBM FHIR Server now supports the ability to compute and store compartment membership values during ingestion. Once stored, these values can help accelerate compartment-related search queries. To use this feature, update the IBM FHIR Server to the latest version and run a reindex operation. See the release notes for Release 4.5.0 for details. The reindex operation reprocesses the resources stored in the database, computing and storing the new compartment reference values. After the reindex operation has completed, add the following configuration element to the relevant tenant fhir-server-config.json file to allow the search queries to use the pre-computed values:
```

Comment:
Don't know if you need to do it now, but I see several ways we refer to the FHIR server throughout the user's guide and I think we should be consistent:
- FHIR server
- IBM FHIR server
- IBM FHIR Server

Reply:
Please use IBM FHIR Server

Reply:
Also, please remove 'now' from 'now supports'

Reply:
Please direct people to FHIRSearch Guide rather than the Release notes.

Reply:
Done